README.md

```diff
@@ -2,15 +2,15 @@
 
 ## Release schedule
 
-| Release  | Status              | Codename    |Initial Release | Active LTS Start | Maintenance Start | End-of-life               |
-| :--:     | :---:               | :---:       | :---:          | :---:            | :---:             | :---:                     |
-| [12.x][] | **Maintenance**     | [Erbium][]  | 2019-04-23     | 2019-10-21       | 2020-11-30        | 2022-04-30                |
-| [14.x][] | **Maintenance**     | [Fermium][] | 2020-04-21     | 2020-10-27       | 2021-10-19        | 2023-04-30                |
-| [16.x][] | **Active LTS**      | [Gallium][] | 2021-04-20     | 2021-10-26       | 2022-10-18        | 2024-04-30                |
-| [17.x][] | **Current**         |             | 2021-10-19     | -                | 2022-04-01        | 2022-06-01                |
-| 18.x     | **Pending**         |             | 2022-04-19     | 2022-10-25       | 2023-10-18        | 2025-04-30                |
-| 19.x     | **Pending**         |             | 2022-10-18     | -                | 2023-04-01        | 2023-06-01                |
-| 20.x     | **Pending**         |             | 2023-04-18     | 2023-10-24       | 2024-10-22        | 2026-04-30                |
+| Release  | Status              | Codename     |Initial Release | Active LTS Start | Maintenance Start | End-of-life               |
+| :--:     | :---:               | :---:        | :---:          | :---:            | :---:             | :---:                     |
+| [12.x][] | **Maintenance**     | [Erbium][]   | 2019-04-23     | 2019-10-21       | 2020-11-30        | 2022-04-30                |
+| [14.x][] | **Maintenance**     | [Fermium][]  | 2020-04-21     | 2020-10-27       | 2021-10-19        | 2023-04-30                |
+| [16.x][] | **Active LTS**      | [Gallium][]  | 2021-04-20     | 2021-10-26       | 2022-10-18        | 2024-04-30                |
+| [17.x][] | **Maintenance**     |              | 2021-10-19     | -                | 2022-04-01        | 2022-06-01                |
+| [18.x][] | **Current**         | [Hydrogen][] | 2022-04-19     | 2022-10-25       | 2023-10-18        | 2025-04-30                |
```

Comment:
I don't think the Codename is added till a release hits LTS

Reply:
Performed requested changes